csharp/src/Google.OpenLocationCode/OpenLocationCodeDecode.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Google.OpenLocationCode
+{
+    public static class OpenLocationCodeDecode
```

Comment:
I did find it slightly confusing how this class which isn't mentioned anywhere becomes available to OpenLocationCode objects - mainly because I'm not familiar with c#. Could you add a comment that explains it?